flake.nix

```diff
@@ -413,9 +421,12 @@
             shell = {
               shellHook = exportOgmiosFixtures;
               packages = [
+                (hsProjectFor system).hsPkgs.ctl-server.components.exes.ctl-server
```

Comment:
I actually think `self.packages.${system}."ctl-server:exe:ctl-server"` might be better